spec/integration/inventory_spec.rb

```diff
@@ -25,24 +25,46 @@ module Hawkular::Inventory::RSpec
     end
   end
 
-  describe 'Inventory', vcr: { decode_compressed_response: true } do
+  describe 'Inventory' do
     before(:all) do
       @creds = {
         username: 'jdoe',
         password: 'password'
       }
       @client = Hawkular::Inventory::InventoryClient.create(credentials: @creds)
-      @state = {
-        hostname: 'localhost.localdomain',
-        feed: nil
-      }
+      options = { decode_compressed_response: true }
+      options[:record] = :all if ENV['VCR_UPDATE'] == '1'
+      VCR.use_cassette('Inventory/Helpers/get_feeds', options) do
+        feeds = @client.list_feeds
+        @state = {
+          hostname: 'dhcp-10-40-1-43.brq.redhat.com',
```

Comment:
When I said 'talks to' - I meant in case of (re-)recording. Because you changed 'localhost' to 'dhcp-...'
But seems the last paragraph you wrote goes into the direction I was thinking as well.